### PR TITLE
Adding health endpoint

### PR DESCRIPTION
**Pull request by edisinovcic**
This is a first version of health endpoint and docker-compose and dockerfile. It's not optimized version but it's quick and dirty first implementation. Please test it and let me know if there are any problems. 

Health endpoint has been added, will soon also add a service that pings it every 60 seconds.

### CODE REVIEW

**Comment by chriseth** [comment]
The target branch seems wrong.

**Comment by edisinovcic** [comment]
Now it's targeting master and I've resolved merge conflicts.

**Comment by chriseth** [comment]
Can you rebase and squash a little?

**Comment by edisinovcic** [comment]
Definitely. :)

File: injector.js [hunk 1]
```diff
@@ -205,5 +205,13 @@ exports.inject = async function(repository, chain, address, files) {
...
+<<<<<<< HEAD
 
 exports.cborDecode = cborDecode;
+=======
+<<<<<<< HEAD
+=======
+
+exports.cborDecode = cborDecode;
+>>>>>>> b70ee4a... Add initial test fixtures and CI
+>>>>>>> 0a02b60... Adding docker-compose for the server and monitor and adding health endpoint to server
```
Comment (cgewecke) [inline injector.js]: There are a few stray merge conflict markers here :)

**Comment by chriseth** [comment]
The issue with the tests looks like a problem on CircleCI's side (password-protected ssh key).

In other projects, we use the policy to never have a merge commit as part of a PR because it complicates the review - shall we adopt this policy also here?

File: injector.js (hunk 1 above)
Comment (edisinovcic) [reply injector.js]: Tnx!

**Comment by edisinovcic** [comment]
We could, how do you solve conflicts when they appear? Do you manually change code that is conflicting without pulling branch that it should be merged into?

**Pull request merged**